base/Enums.jl

```diff
@@ -68,6 +68,38 @@ To list all the instances of an enum use `instances`, e.g.
 julia> instances(Fruit)
 (apple::Fruit = 1, orange::Fruit = 2, kiwi::Fruit = 3)
 ```
+
+`Base.Enums.basetype` function returns `BaseType`.
+
+```julia
+julia> @enum Vegetable::UInt8 pea carrot broccoli
+
+julia> Vegetable
+Enum Vegetable:
+pea = 0x00
+carrot = 0x01
+broccoli = 0x02
+
+julia> Base.Enums.basetype(Vegetable)
```

Comment:
Can't we just use the enum already defined?

Reply:
No because that's for showing a `BaseType` which is different from default one.

Reply:
This docstring is becoming very long so we need to think about ways of shortening it. I think adding a docstring to `Enums.basetype` and just referring to it here is a good idea?